### PR TITLE
Hydrology

### PR DESCRIPTION
**Pull request by tplshams**

### CODE REVIEW

File: book/tutorials/Hydrology/Hackweek.ipynb
Comment (scottyhq) [inline book/tutorials/Hydrology/Hackweek.ipynb:1]: I recommend changing `/home/jovyan/shared/ICESat-2-Hackweek/Hydrology` -> `/tmp/Hydrology` (not everyone has write access to `shared/`

**Review by scottyhq — CHANGES_REQUESTED** [review]
Note: failing checks
https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/actions/runs/5767906781/job/15638272448?pr=74

`Ensure Clean Notebook` is a reminder to commit a notebook *without* outputs. So `Restart Kernel and Clear all Outputs` from the file menu

**Comment by jomey** [comment]
Here some spelling errors identified by the QA/QC automation:
```
Error: ./book/tutorials/Hydrology/Hackweek.ipynb:28: respresented ==> represented
Error: ./book/tutorials/Hydrology/Hackweek.ipynb:53: demostration ==> demonstration
Error: ./book/tutorials/Hydrology/Hackweek.ipynb:2155: ONy ==> only, on, one
Error: ./book/tutorials/Hydrology/Hackweek.ipynb:2203: Te ==> The, Be, We, To
Error: ./book/tutorials/Hydrology/Hackweek.ipynb:2203: bu ==> by, be, but, bug, bun, bud, buy, bum
Error: ./book/tutorials/Hydrology/Hackweek.ipynb:2271: Ths ==> The, This
Error: ./book/tutorials/Hydrology/Hackweek.ipynb:2271: bU ==> by, be, but, bug, bun, bud, buy, bum
```

Feel free to add words to ignore to this list:
https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/blob/tplshams-patch-1/.codespellignore

**Comment by jomey** [comment]
@tplshams - I updated the notebook to be committed as the rendered version to overcome the challenge of not having access to the CryoHub from our GitHub actions. There were also some minor changes needed the directories to make it run.

File: .github/workflows/qaqc.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       with:
         check_filenames: true
         check_hidden: false
-        skip: '*.js,qaqc.yml,acp_lidar_box.geojson'
+        skip: '*.js,qaqc.yml,acp_lidar_box.geojson,.codespellignore'
```
Comment (jomey) [inline .github/workflows/qaqc.yaml:35]: @scottyhq - I am suspicious that the QA/QC workflow didn't run after this change. Let's keep an eye on this

**Pull request merged**